e2e/celestia-app/app.toml

```diff
@@ -0,0 +1,252 @@
+# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+###############################################################################
+###                           Base Configuration                            ###
+###############################################################################
```

Comment:
its definitely not worth prioritizing, however in other tests we actually generate the default for this each time we run the test, and then modify any test specific values before saving.

Reply:
since we're not `init`ing the nodes in e2e, we need this file to specify the min gas price, which is becoming mandatory to have before starting a node